docs/integrations/ember.rst

```diff
@@ -1,24 +1,88 @@
 Ember
 =====
 
+To use Sentry with your Ember application, you will need to use both Raven.js (Sentry's browser JavaScript SDK) and the Raven.js Ember plugin.
+
+On its own, Raven.js will report any uncaught exceptions triggered from your application. For advanced usage examples of Raven.js, please read :doc:`Raven.js usage <../usage>`.
+
+Additionally, the Raven.js Ember plugin will catch any Ember-specific exceptions reported through Ember's `onerror <https://guides.emberjs.com/v2.3.0/configuring-ember/debugging/#toc_implement-an-ember-onerror-hook-to-log-all-errors-in-production>`_. hook
+and any `RSVP promises <https://guides.emberjs.com/v2.3.0/configuring-ember/debugging/#toc_errors-within-an-code-rsvp-promise-code>`_ that would otherwise be swallowed.
+
 Installation
 ------------
 
-Start by adding the ``raven.js`` script tag to our page. You'll want to position it
-after you load all other external libraries (like jQuery), but before your code.
+Raven.js and the Raven.js Ember plugin are distributed using a few different methods.
+
+Using our CDN
+~~~~~~~~~~~~~
+
+For convenience, our CDN serves a single, minified JavaScript file containing both Raven.js and the Raven.js Ember plugin. It should be included **after** Ember, but **before** your application code.
+
+Example:
 
 .. sourcecode:: html
 
+    <script src="http://builds.emberjs.com/tags/v2.3.1/ember.prod.js"></script>
     <script src="https://cdn.ravenjs.com/2.1.0/ember/raven.min.js"></script>
+    <script>Raven.config('___PUBLIC_DSN___').install();</script>
+
+Note that this CDN build auto-initializes the Ember plugin.
+
+Using package managers
+~~~~~~~~~~~~~~~~~~~~~~
+
+Pre-built distributions of Raven.js and the Raven.js Ember plugin are available via both Bower and npm for importing in your ``ember-cli-build.js`` file.
```

Comment:
Pretty sure. I wouldn't be able to document a custom build tool here very well anyway.